### PR TITLE
fix: change nlohmann-json import path to be absolute

### PR DESCRIPTION
**Pull request by themarwhal**
Signed-off-by: Marie Bremner <marwhal@fb.com>

## Summary
Fixing up the json include path to use the full path, not just json.hpp. (It works either way)

This change also includes clean up work to remove the unnecessary symlink from vagrant/docker.

## Test Plan
All unit tests
Built AGW and tested in phyu2

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Review by rdefosse — APPROVED** [review]

File: lte/gateway/docker/mme/Dockerfile.rhel8 [hunk 1]
```diff
@@ -344,8 +344,7 @@ RUN cd json && \
     cd _build && \
     cmake3 -G "Unix Makefiles" -DCMAKE_BUILD_TYPE=Release .. && \
     make -j`nproc` && \
-    make install && \
-    cp /usr/local/include/nlohmann/json.hpp /usr/local/include/
```
Comment (rdefosse) [inline lte/gateway/docker/mme/Dockerfile.rhel8:348]: @themarwhal  Marie

This modification is not tested in the OAI CI pipeline. Only when your PR is `merged`. But it should be OK.

Ping me when it is merged. I will also have to recreate a base image with this modification.

Regards,
Raphael

**Review by electronjoe — APPROVED** [review]
Thanks for the reduction of work involving the phrase "hacks" =D

File: lte/gateway/docker/mme/Dockerfile.rhel8 (hunk 1 above)
Comment (themarwhal) [reply lte/gateway/docker/mme/Dockerfile.rhel8:348]: Is there a way to ensure nothing breaks before merging?

Comment (rdefosse) [reply lte/gateway/docker/mme/Dockerfile.rhel8:348]: Let me try manually

Comment (rdefosse) [reply lte/gateway/docker/mme/Dockerfile.rhel8:348]: @themarwhal Marie

OK for me. You can merge!

**Review by ardzoht — APPROVED** [review]

**Pull request merged**